post-office-stub/src/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "F2F",
+  "version": "1.0.0",
+  "description": "OneLogin GovNotify stub",
+  "repository": "https://github.com/alphagov/di-ipv-cri-f2f-api",
+  "author": "GDS",
+  "license": "MIT",
+  "dependencies": {
+    "@aws-lambda-powertools/logger": "1.5.1",
+    "@aws-lambda-powertools/metrics": "1.5.1",
+    "@aws-sdk/client-dynamodb": "3.150.0",
```

Comment:
Think there's a few packages in here that we aren't using in the stub itself, could we remove them? Means less package upgrades to manage!

Reply:
Fixed